res/css/views/rooms/_ReplyPreview.pcss

```diff
@@ -20,6 +20,7 @@ limitations under the License.
     background: $background;
     max-height: 50vh;
     overflow: auto;
+    margin: 0 0 0 -8px;
```

Comment:
This will shift the reply preview left by 8 pixels in all scenarios. 
That might create a visual regression in the main room timeline too. Could you add some screenshots for that context too so we can validate whether it looks adequate.

We also tend to use `$spacing-8` rather than hardcoding a value. `calc(-1 * $spacing-8)` should achieve the same result here